nbconvert/filters/markdown_mistune.py

```diff
@@ -104,15 +104,20 @@ def header(self, text, level, raw=None):
         html = super(IPythonRenderer, self).header(text, level, raw=raw)
         return add_anchor(html)
 
+    def escape_lt(self,text):
+        return text.replace('<','&lt;')
```

Comment:
I didn't find any bug triggered by ">". But it is a good idea to escape all these special characters.